src/views/Portal/components/ApplicationForm/index.view.tsx

```diff
@@ -199,6 +200,11 @@ const ApplicationForm: React.FC = () => {
       } else {
         setServerErrors([])
       }
+      if (err && err.response && err.response.status === 401) {
```

Comment:
Uh oh.. How often does the token expire?

Reply:
It expires after 24 hours which is fairly forgiving. I've only seen ~3 instances of users getting 401s according to our loggers so it's not a huge issue but definitely something worth clarifying over the last day of applications.